WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerAdapter.kt

```diff
@@ -12,7 +13,11 @@ import org.wordpress.android.ui.mediapicker.MediaPickerUiItem.Type
 import org.wordpress.android.ui.mediapicker.MediaPickerUiItem.VideoItem
 import org.wordpress.android.util.image.ImageManager
 
-class MediaPickerAdapter internal constructor(imageManager: ImageManager) : Adapter<ThumbnailViewHolder>() {
+class MediaPickerAdapter internal constructor(
+    private val imageManager: ImageManager,
+    private val coroutineScope: CoroutineScope
+) :
+        Adapter<ThumbnailViewHolder>() {
```

Comment:
🔍 Nitpick: It seems this extra line break is not necessary.

Reply:
Fixed in 99c9982eb0